docker/training/build-hadoop.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
This file looks exactly the same as the docker/inference/build-hadoop.sh. Do we need to have two copies of the same file? Can we just use one ?

(likewise for install-hadoop.sh)

Reply:
That is correct. Since you currently require different subdirectories for making these, there is no other way. I would suggest putting all dockerfiles into the same subfolder. Then  you won't need the files twice.